app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -489,6 +510,7 @@ class BrowserTabFragment :
         }
 
         addTextChangedListeners()
+        resumeWebView()
```

Comment:
How about only calling this if voice search is available? It's not needed otherwise.

Reply:
As we talked about awhile ago, there shouldn't be any issue on this. resumeWebView() only matters to be called manually when the user have opened the voice search feature. Else, if the user doesn't support voice search OR just don't use the voice search even when it is available, webview is automatically resumed.